daemon/container.go

```diff
@@ -330,5 +330,9 @@ func (daemon *Daemon) verifyContainerSettings(platform string, hostConfig *conta
 	}
 
 	// Now do platform-specific verification
-	return verifyPlatformContainerSettings(daemon, hostConfig, config, update)
+	platformWarns, err := verifyPlatformContainerSettings(daemon, hostConfig, config, update)
+	if hostConfig.NetworkMode.IsHost() && len(hostConfig.PortBindings) > 0 {
+		platformWarns = append(platformWarns, "Setting published IPs is useless when network mode is set to host")
```

Comment:
LOL, had to laugh a bit about your choice of wording :smile:

![useless](https://user-images.githubusercontent.com/1804568/32844776-830499c6-ca23-11e7-92d9-cc1886238fa6.gif)

I'll try to come up with an alternative 👍